js/components/app.js

```diff
@@ -4,23 +4,41 @@ import getMuiTheme from 'material-ui/styles/getMuiTheme';
 import darkBaseTheme from 'material-ui/styles/baseThemes/darkBaseTheme';
 import Thermoscope from './thermoscope';
 import injectTapEventPlugin from 'react-tap-event-plugin';
+import LabQuest2 from 'sensor-labquest-2-interface';
 
 import '../../css/app.less';
+import '../../css/sensor-connect.less';
 
 // Required by Material-UI library.
 injectTapEventPlugin();
 
+// For the LabQuest2 sensor
+var connect = function() {
+  LabQuest2.connect(document.getElementById("sensorIP").value);
+  document.getElementById("sensorConnectionStatus").innerHTML = "Connecting...";
+  LabQuest2.on('connected', function(){
+    document.getElementById("sensorConnectionStatus").innerHTML = "Connected.";
```

Comment:
Hm, this doesn't really fit React app. We could implement such logic inside component using React state.